src/main/java/hive/pokedex/util/FillNullValues.java

```diff
@@ -0,0 +1,28 @@
+package hive.pokedex.util;
+
+import org.springframework.beans.BeanUtils;
+import org.springframework.beans.BeanWrapperImpl;
+
+import java.beans.PropertyDescriptor;
+import java.util.HashSet;
+import java.util.Set;
+
+public final class FillNullValues {
+  public static void copyProperties(final Object destiny, final Object origin) {
```

Comment:
Why (destiny,origin) instead of the (origin,destiny)? Also, source might be better than origin.